mod/pages/views/default/forms/pages/edit.php

```diff
@@ -13,6 +13,11 @@
 $entity = elgg_extract('entity', $vars);
 $parent_guid = elgg_extract('parent_guid', $vars);
 
+$can_change_access = true;
+if ($entity instanceof ElggPage && $user instanceof ElggUser) {
+	$can_change_access = ($user->isAdmin() || $entity->owner_guid === $user->guid);
```

Comment:
why not $entity->getOwnerEntity()->canEdit()?

Reply:
this is the logic used in the action, so keeping it the same

Reply:
you can also change it in the action...